tests/test_es_connection.py

```diff
@@ -116,9 +116,9 @@ def test_result_history(self, es):
         assert es.put_object(self.uuid(check4), check4)
         assert es.refresh_index()
         assert es.get_size() == 4
-        res = es.get_result_history('page_children_routes', 0, 25)
+        res = es.get_result_history('page_children_routes', 0, 25, sort=None)[0]
         assert len(res) == 3
-        res = es.get_result_history('check_status_mismatch', 0, 25)
+        res = es.get_result_history('check_status_mismatch', 0, 25, sort=None)[0]
```

Comment:
What does `sort=None` do here? Does the return value from `es.get_result_history` already come with some interesting sort that makes `[0]` access that (where sorting might perturb it), or does it make `[0]` implement a random selection because the returned data is in fact unsorted?